docs/index.md

```diff
@@ -89,7 +89,7 @@ You'll need to set up new Shopify API credentials to connect Theme Kit to your s
 
 1. In the store's Shopify admin, click **Apps**.
 1. Near the bottom of the page, click on **Manage private apps**.
-1. If you see a notice that "Private app development is disabled", then click "Enable private app development".
+1. If you see a notice that "Private app development is disabled", then click "Enable private app development". (**Note:** Only the Store Owner can enable private app development. More information available [here](https://help.shopify.com/en/manual/apps/private-apps#enable-private-app-development-from-the-shopify-admin))
```

Comment:
Minor: Looks like we don't usually capitalize Store Owner (at least not on https://help.shopify.com/en/manual/apps/private-apps#enable-private-app-development-from-the-shopify-admin )

Reply:
+1 to Andy's point. Also, in general we try to use more contextual links than "here":

```suggestion
1. If you see a notice that "Private app development is disabled", then click "Enable private app development". (**Note:** Only the store owner can enable private app development. Learn more about [enabling private apps](https://help.shopify.com/en/manual/apps/private-apps#enable-private-app-development-from-the-shopify-admin)).
```